tests/test_table.cc

```diff
@@ -62,6 +62,8 @@ void test_strtable(const vector<std::string>& keys, uint32_t num_to_insert) {
       upb_strtable_next(&iter)) {
     const char *key = upb_strtable_iter_key(&iter);
     std::string tmp(key, strlen(key));
+    std::string tmp2(key, upb_strtable_iter_keylength(&iter));
+    ASSERT(tmp == tmp2);
```

Comment:
Maybe `ASSERT(strlen(key) == upb_strtable_iter_keylength(&iter))` here to avoid extra copy and data comparison?